tasks/eslint.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.config('eslint', {
+    options: {
+      eslintrc: '.eslintrc',
+      src: ['Gruntfile.js', 'tasks/*.js', 'config/**/*.js', 'node/**/*.js']
+    },
```

Comment:
Sorry, the longer I stare at line 9 above, the more it weirds me out that we have `src` defined in `options`.

Can we:
1. Move that `src` above into a new target named "app",
2. Rename "app" target below (line 11) to something like "client"